v2/contribute/decisions/session/0027-use-standard-jwt-verification-lib-on-BE.mdx

```diff
@@ -0,0 +1,29 @@
+---
+id: "0027"
+title: Use a standard JWT verification lib on the BE
+hide_title: true
+---
+
+import DecisionHeader from "/src/components/decisionLogs/DecisionHeader"
+import ArgumentPro from "/src/components/decisionLogs/ArgumentPro"
+import ArgumentCon from "/src/components/decisionLogs/ArgumentCon"
+import ArgumentNeut from "/src/components/decisionLogs/ArgumentNeut"
+
+# Use a standard JWT verification lib on the BE
+
+<DecisionHeader status="proposed" lastUpdate="2022-12-06" created="2022-12-06" deciders={["rishabhpoddar", "porcellus"]} proposedBy={["porcellus"]} />
+
+## Context and Problem Statement
+
+Since our access tokens will match standard JWTs, we can switch to using a jwt verification library with key caching instead of our own implementation.
+
+## Considered Options
+
+* Keep using our own implementation
+* Switch to an external library
+
+## Decision Outcome
+
+We should switch to an external library
+- Our code gets smaller
+- We use the same method we will recommend to people if they aren't using our SDKs
```

Comment:
I think we actually need both:
- The backend handling auth will call the core JWKs endpoint.
- The FE calls the JWKs endpoint exposed by the backend (I'm not sure how much this would be used)
- Other backends (microservices) could call either the core or the other backend JWKs endpoint. 
- Otherwise, we'd require exposing the Core to the internet, right now, it can be just a locally run thing (e.g., inside the docker on the server providing the backend) or something you connect to through VPN or some local network.